### ISSUE TITLE
Spell Descriptions

### ISSUE THREAD

**Issue by AceDashDog** [danielloeblein/oathmark-army-builder#4]
Hi! Fabulous software. Not an issue but a question. I added a couple of spell descriptions in spellist.json but they don't appear on the print, is there something else I need to do?

        {
            "name": "Dispel",
            "castingNumber": 8,
            "description": "The spellcaster may cast this spell on any unit. All ongoing effects from all spells cast on the target unit or figure are cancelled. This spell has no effect on spells that have a one-off effect, such as Heal"
        },
Cheers

**Comment by danielloeblein** [comment]
Have a look at the compendium.ts, linea 58 & 59:
Change 
        this.compendiumContainer.style.display = 'none';//this.isCompendiumContainerFilled ? 'block' : 'none';
        this.compendiumNav.style.display = 'none';//this.isCompendiumContainerFilled ? 'block' : 'none'; 
to
        this.compendiumContainer.style.display = this.isCompendiumContainerFilled ? 'block' : 'none';
        this.compendiumNav.style.display = this.isCompendiumContainerFilled ? 'block' : 'none'; 

and run bundle again. That should do the trick.

**Comment by AceDashDog** [comment]
Thanks for coming back! I did that, and it doesn't seem to have changed anything. Do I need to compile it or something?

A 'simple' workaround would be to make Notes an editable and saved field, like Name. Could put the caster's spells in that :)
 
Cheers


 

--



Hugh Foster 

***@***.***

http://www.ace-dog.com 

A cynic is a blackguard whose faulty vision sees things as they are, not as they ought to be. (Ambrose Bierce)

--



Sunday, February 26, 2023, 7:03:52 PM, you wrote:

**Comment by danielloeblein** [comment]
Yeah, you need to compile via npm (https://en.wikipedia.org/wiki/Npm_(software) / installable via node: https://nodejs.org/en/), the scripts name is bundle

**Comment by AceDashDog** [comment]
Urm. OK, I'll try, I have never worked in JS; my programming was on mainframes in the DOS era....
